pipeline/metadata/5.3.yaml

```diff
@@ -1147,6 +1147,20 @@ suites:
       - rgw
       - stage-3
 
+  - name: "test-rgw-ingress-with-and-without-ssl-tier-2"
+    suite: "suites/pacific/rgw/tier-2_rgw_ingress_with_ssl.yaml"
+    global-conf: "conf/pacific/rgw/tier-1_rgw_cephadm.yaml"
+    platform: "rhel-8"
+    rhbuild: "5.3"
+    inventory:
+      openstack: "conf/inventory/rhel-8-latest.yaml"
+    metadata:
+      - schedule
+      - tier-2
+      - openstack-only
+      - rgw
+      - stage-4
```

Comment:
Can be added as stage-1
if this will be added as a part of 6.0 run then there also can be added as stage-1

Reply:
done